qemu/tests/cfg/balloon_small.cfg

```diff
@@ -0,0 +1,18 @@
+- balloon_small: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4, Unix, livecd
+    type = balloon_check
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    small_value_check = yes
+    balloon_type = evict
+    test_tags = "evict"
+    kill_vm_on_error = yes
+    Windows:
+        expect_memory = 10
```

Comment:
What's the reason to distinguish Linux and Windows in the two different kind of ways?

Reply:
if reboot windows guest after set to a very memory, guest memory can be reduced to quite small value, which lead to guest almost freeze, login will always fail.